coding-concepts.js

```diff
@@ -1,56 +1,42 @@
-// ASSESSMENT 1: Coding Conceptual Questions
-
+//ASSESSMENT 1: Coding Conceptual Questions --
```

Comment:
Arrays and strings can be confusing in the beginning. I suggest reviewing the javascript introduction and the arrays intro section in the syllabus.